### ISSUE TITLE
How can I reproduce TNT-Meetingroom result?

### ISSUE THREAD

**Issue by DerKleineLi** [NVlabs/neuralangelo#14]
Hi there, thanks for publishing the code.

I'm trying to reproduce the result of Meetingroom in the tanks and temples dataset. I run `projects/neuralangelo/scripts/convert_tnt_to_json.py` to process the data and use the following config:
```yaml
_parent_: projects/neuralangelo/configs/base.yaml

model:
    object:
        sdf:
            mlp:
                inside_out: True   # True for Meetingroom.
            encoding:
                coarse2fine:
                    init_active_level: 8
    appear_embed:
        enabled: True
        dim: 8

data:
    type: projects.neuralangelo.data
    root: .../data/TNT/Meetingroom
    num_images: 371  # The number of training images.
    train:
        image_size: [835,1500]
        batch_size: 1
        subset:
    val:
        image_size: [300,540]
        batch_size: 1
        subset: 1
        max_viz_samples: 16
```

However, I only get a PSNR around 22 at iteration 300k. Do you have any suggestions to improve it?

Thank you!

**Comment by mli0603** [comment]
Hi @DerKleineLi 

Thank you for your interest in the project. 

The configs look correct to me. How many GPUs did you use?

**Comment by DerKleineLi** [comment]
I use 1 rtx a6000 to train, here's the command I used:
```bash
EXPERIMENT=tnt_meetingroom
GROUP=meetingroom
NAME=original
CONFIG=projects/neuralangelo/configs/meetingroom.yaml
GPUS=1  # use >1 for multi-GPU training!
python -m torch.distributed.run --nproc_per_node=${GPUS} train.py \
    --logdir=/cluster/angmar/hli/neuralangelo_outputs/${GROUP}/${NAME} \
    --config=${CONFIG} \
    --show_pbar \
    --wandb \
    --wandb_name neuralangelo
```

**Comment by mli0603** [comment]
Hi @DerKleineLi 

For TNT, we used a batch size of 16 (distributed on 8 GPUs). This will speed up the convergence and improve the quality. The details can be found in Appendix A of the paper.

**Comment by DerKleineLi** [comment]
@mli0603 Thanks for the reply!

I noticed that some default parameter in the code does not match the paper as well. They are num_layers of the sdf mlp and the weight decay rate. Do you recommend using the parameter in the code or in the paper?

**Comment by mli0603** [comment]
Hi @DerKleineLi 

Nice catch! The weight decay value is a typo and has been fixed in this commit (https://github.com/NVlabs/neuralangelo/commit/e398a3bdc841448b75ebbed64935ac7499fcd82d). 

The # of mlp layers actually is correct. The paper refers to the latent MLP layers when compared to NeuS (1 layer vs 8 layers), which ignores the output projection layer.

**Comment by chenhsuanlin** [comment]
Closing due to inactivity, please feel free to reopen if there are further issues!
